Sources/Extensions/UIColor.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  UIColor.swift
+//  MessageKit
+//
+//  Created by Vlada Radchenko on 9/30/19.
+//  Copyright © 2019 MessageKit. All rights reserved.
+//
+
+import Foundation
+
+extension UIColor {
```

Comment:
Any reason these are in a separate file rather than adding them to UIColor+Extensions.swift ?

Reply:
He probably didn't see it. Can make another PR to move these when I get free time, or it's even better if @Vlada31R could do it